src/types/TestImporter.ts

```diff
@@ -0,0 +1,22 @@
+import { MatchCategory } from './Match/MatchCategory';
+import { Match } from './Match/Match';
+
```

Comment:
When using IDE completion, the below is what it automatically generates in IntelliJ. Currently I have to manually type it - which through investigation is the root cause of this issue. Any ideas?